README.md

```diff
@@ -128,11 +149,10 @@ Then run this line in your terminal
 
 ## 🔭 Future Features <a name="future-features"></a>
 
-- [ ] **[More quaries]**
+- [ ] **[Implement sort enumerable]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-
 ## 🤝 Contributing <a name="contributing"></a>
```

Comment:
- [ ] 👋 Hey there! I've noticed that you've implemented a custom link for your project's license 📜, but it appears to be broken 🚫❌. The good news is, I see that you have a LICENSE file 📁 right there in the root of your project 🌳.
So, why not switch things up a bit and make life easier for everyone? 🤔 Let's update the link to point directly to this trusty LICENSE file 🗺️. It'll ensure that all users can easily access and understand the terms and conditions 📖 without any hassles. 🤩👍

![Screenshot from 2023-11-03 14-05-57](https://github.com/sagieramos/my_enumerable/assets/112261237/bf78fe80-6c0d-4c4f-8715-cadf137ce343)

```suggestion
This project is [MIT](./LICENSE) licensed.
```